src/api.js

```diff
@@ -1,56 +1,67 @@
 //GET requests
 export const sleepDataAPI = fetch("http://localhost:3001/api/v1/sleep")
   .then(response => response.json())
+  .catch(err => alert('Sorry, but there was an error. Please try again.'))
```

Comment:
I wish you kept the "Not Enough Sauce" Alert...but nice work on adding these here.